tools/generators/react-library/index.ts

```diff
@@ -0,0 +1,139 @@
+import * as path from 'path';
```

Comment:
Can we have an offline talk about this? I would love to maximize the reusability of this generator for the [contrib repo](https://github.com/microsoft/fluentui-contrib)